src/app/src/mainwindow.cpp

```diff
@@ -263,9 +298,15 @@ void MainWindow::on_durationEdit_valueChanged(double value) {
         spdlog::error("Capture is set to less than 1 frame, fps: {}, duration: {}", value, m_duration);
         ui.frameRateEdit->setStyleSheet("background-color: red");
         ui.durationEdit->setStyleSheet("background-color: red");
-    } else {
+    }else if (!available_space_in_drive(m_path,value,m_duration)){
```

Comment:
is `m_path` the drive path or the full path for acquisition? Either way we should be able to use it in the tooltip message below instead of hardcoding the `E:\\` drive value